build/build_suite_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"testing"
 
 	"github.com/gophercloud/gophercloud"
+	starlingxv1 "github.com/wind-river/cloud-platform-deployment-manager/api/v1"
```

Comment:
why use startlingxv1? can we use dm-api-v1?

Reply:
FOllowed as per existing code refs